src/uPlot.js

```diff
@@ -1681,8 +1681,17 @@ export default function uPlot(opts, data, then) {
 			_y = e.clientY - rect.top;
 		}
 		else {
-			_x = plotWidCss * (_x/_w);
-			_y = plotHgtCss * (_y/_h);
+			// check for value based syncs in the src plot
+			let [xKey, yKey] = src.cursor.sync.scales;
```

Comment:
unless there's a reason why these config opts would differ between publisher (src) and subscriber (self), let's just pull this off of the `syncOpts`.